fahrschein/src/main/java/org/zalando/fahrschein/domain/SubscriptionRequest.java

```diff
@@ -32,16 +32,18 @@ public String toString() {
     private final Set<String> eventTypes;
     private final String consumerGroup;
     private final Position readFrom;
+    private final Set<Cursor> initialCursors;
 
-    public SubscriptionRequest(String owningApplication, Set<String> eventTypes, String consumerGroup, Position readFrom) {
+    public SubscriptionRequest(String owningApplication, Set<String> eventTypes, String consumerGroup, Position readFrom, Set<Cursor> initialCursors) {
         this.owningApplication = owningApplication;
         this.eventTypes = unmodifiableSet(eventTypes == null ? emptySet() : new HashSet<>(eventTypes));
         this.consumerGroup = consumerGroup;
         this.readFrom = readFrom;
+        this.initialCursors = initialCursors;
```

Comment:
`SubscriptionRequest` is supposed to be immutable, so it would be better to create an unmodifiable copy. I'd also prefer a `List` instead of a `Set`, `Cursor` has a `hashCode` implementation, but I consider that a mistake as it's only used in tests.